test/gtest/ai_heuristics.hpp

```diff
@@ -0,0 +1,15 @@
+#include <miopen/miopen.h>
+#include "../tensor_holder.hpp"
+#include <gtest/gtest.h>
+#include <miopen/solver.hpp>
+#include "get_handle.hpp"
+#include <gtest/cba.hpp>
+#include <miopen/conv/heuristics/ai_heuristics.hpp>
```

Comment:
I think it is enough to include here header files for `ConvTestCase`, `miopen::conv::Direction`, `miopenDataType_t` and `miopenTensorLayout_t`.